spec/features/right_word.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+describe 'test with wrong word' do
+
+  it 'right word' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.